imagehash/__init__.py

```diff
@@ -33,211 +33,243 @@
 
 from PIL import Image
 import numpy
-#import scipy.fftpack
-#import pywt
+
+
+__version__ = '3.1.0.dev0'
 
 
 def _binary_array_to_hex(arr):
```

Comment:
All of the following changes (apologies) are for pep8 compliance (which is a very good thing indeed!)